packages/atlas/src/providers/personalData/store.ts

```diff
@@ -15,6 +15,8 @@ export type PersonalDataStoreState = {
   cinematicView: boolean
   cookiesAccepted?: boolean
   reactionPopoverDismissed: boolean
+  playbackRate: number
+  playNext: boolean
```

Comment:
Could we maybe call this `autoPlayNext` or something else? Current name is a bit of mystery